setup.sh

```diff
@@ -3,15 +3,17 @@ GREEN='\033[0;32m'
 CYAN='\033[0;36m'
 NC='\033[0m' # No Color
 
-
-printf "Enter license key(https://uibakery.io/get-an-on-prem-version-of-ui-bakery?utm_source=install_sh):\n"
```

Comment:
This way link is can be clicked

Reply:
I was able to click it even in my way :D

Reply:
![image](https://user-images.githubusercontent.com/17410089/181709008-f8f16b93-cb04-4c1b-ab6e-c0ba6c25dea2.png)
I can't click on this link. Let's just print the link in terminal